docs/gallery-publishing-howto.md

```diff
@@ -0,0 +1,69 @@
+# Azure AI Starter Template
+
+> [!IMPORTANT]
+> This file provides guidelines to complete setup after scaffolding and to validate your codebase. Feel free to remove this file.
+
+## Standards and conventions
+
+For a complete set of standards and conventions that will guarantee successful template validation, [visit this repository](https://github.com/Azure-Samples/azd-template-artifacts)
+
+## Getting started
+
+Before you get started please make sure to have the [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/overview) installed. You may also need to meet all requirements in the [readme.md](./README.md) file.
+
+### Step 1 - Scaffolding
+
+To proceed with scaffolding using the terminal, change local directory to the desired one, 
+
+`cd [my-project-directory]`
+
+and run
+
+`azd init --template [name-of-template]`
```

Comment:
What about the option to start with your own project repo (not a template) with `azd init`?

Reply:
It does only fit some scenarios, right? And we can't generate the MI example (which will be an integration for the 2nd iteration), scripts and some other files. This is why we required a new command. We can discuss offline.